common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -265,7 +265,7 @@ const SearchTabs: FunctionComponent<Props> = ({
             sortOrder={undefined}
             linkResolver={params => {
               const queryWithSource = propsToQuery(params);
-              const { source = undefined, ...queryWithoutSource } = {
+              const { ...queryWithoutSource } = {
```

Comment:
Are you sure we don't need to delete the source property here? I thought we had to otherwise it get's exposed in the URL?

Reply:
ah, possibly. I'll put it back